src/components/DisplaySimpleProperty/DisplaySimpleProperty.tsx

```diff
@@ -22,6 +22,7 @@ export const DisplaySimpleProperty = ({
   wrap,
   ...props
 }: DisplaySimplePropertyProps) => {
+  const shouldRenderElement = loading || (label && value);
```

Comment:
Ah my bad I forgot that logic. Can you return early here with this logic?

Reply:
good call, changed